src/main/scala/com/typesafe/sbt/SbtMultiJvm.scala

```diff
@@ -35,7 +35,7 @@ object MultiJvmPlugin extends AutoPlugin {
 
     val multiJvmJavaCommand = TaskKey[File]("multi-jvm-java-command")
 
-    val jvmOptions = TaskKey[Seq[String]]("jvm-options")
+    val jvmOptions = TaskKey[Seq[String]]("jvm-options") // TODO: shouldn't that be regular `javaOptions`?
```

Comment:
hmm perhaps it could but then done in other scope as this one is for the forked vm i think?

Reply:
Yea, this makes sense. All of the settings are defined under the MultiJvm config (`inConfig(MultiJvm)(...)`) so the plugin should reuse as much keys as possible, as per [sbt plugin guidelines](http://www.scala-sbt.org/1.0/docs/Plugins-Best-Practices.html).

Also, there are some other keys that do not have miltiJvm as a prefix. So all of the current keys should either have the multijvm prefix added to them or removed altogether in favor of already existing keys.